appcues/src/main/java/com/appcues/ui/extensions/StyleComponentExt.kt

```diff
@@ -176,6 +176,13 @@ internal fun ComponentStyle.getHorizontalAlignment(): Alignment.Horizontal {
 internal fun ComponentStyle?.getBoxAlignment(): Alignment {
     if (this == null) return Alignment.Center
 
+    return getBoxAlignment(horizontalAlignment, verticalAlignment)
+}
+
+internal fun getBoxAlignment(
```

Comment:
ComponentStyle.getBoxAlignment used by style.horizontal style.vertical Alignment

we needed a method that could receive both parameters because of backgroundImage.horizontal backgroundImage.verticalAlignment